sensorserviceutils/sensorserviceutils.go

```diff
@@ -454,10 +508,65 @@ func (p parser) extractActiveSensorInfo() ([]error, []error) {
 	return p.parsingErrs, p.sensorErrs
 }
 
+// organizeSensorInfoForActiveConn is a helper function that specifically
+// handle the sensors' information parsed for each active connection.
+// A special case it handles is when there are multiple sensors being activated
+// by one active connection, then multiple active connection objects will be
+// created.
```

Comment:
What does it mean to have multiple sensors being activated?

Reply:
So as shown in the comment, there is the case where one active connection records the active subscription of multiple sensors subscribing to the same package A. This change accommodates this case by creating one active connection object for each sensor subscribing package A.